src/main/scala/org/constellation/gossip/GossipService.scala

```diff
@@ -57,7 +58,17 @@ abstract class GossipService[F[_]: Parallel, A](
       )
       sequenceToRetry = sequence.dropWhile(node => !failureNode.contains(node))
       clients <- sequenceToRetry.toList.traverse(getClientMetadata)
-      _ <- clients.traverse(spreadFn(_, message)) // TODO: Force trigger peer healthcheck if it fails
+      _ <- clients.parTraverse { client =>
+        generatePathId.flatMap(
```

Comment:
just a design, but it looks better like this:
```scala
generatePathId.flatMap { pathId =>
 ...
 }
```
than this
```scala
generatePathId.flatMap(
  pathId =>
    ...
 )
```

Reply:
Agree I like it more as well, fixing.